PlacenoteSDK.podspec

```diff
@@ -17,7 +17,7 @@ Pod::Spec.new do |s|
     s.license           = { :type => 'Placenote EULA', :file => 'LICENSE' }
 
     s.platform          = :ios, '11.0'
-    s.source            = { :git => 'https://github.com/Placenote/PlacenoteSDK-CocoaPod.git', :tag => 'v1.6.12' }
+    s.source            = { :git => 'https://github.com/Placenote/PlacenoteSDK-CocoaPod.git', :tag => 'v1.7' }
     s.swift_version     = '3.2'
```

Comment:
does this need to be changed to 4 ? I remember we were having some issues with Swift version in the pod. Is this anything related?